lighthouse-treemap/test/treemap-test-pptr.js

```diff
@@ -141,4 +141,49 @@ describe('Lighthouse Treemap', () => {
       expect(optionsInPage.lhr.requestedUrl).toBe(options.lhr.requestedUrl);
     });
   });
+
+  describe('renders correctly', () => {
+    it('correctly shades coverage of gtm node', async () => {
+      await page.goto(`${treemapUrl}?debug`, {
+        waitUntil: 'networkidle0',
+        timeout: 30000,
+      });
+
+      await page.click('#view-mode--unused-bytes');
+      await page.waitForSelector('.lh-treemap--view-mode--unused-bytes', {visible: true});
```

Comment:
```suggestion
      await page.waitForSelector('.lh-treemap--view-mode--unused-bytes');
```

visible doesn't seem necessary?

Reply:
yah. no big deal to me. couldn't tell you the tradeoffs.